postgresql-containers/bin/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# pgdata/ is mounted to the pg database container with rw permissions
+recovery_file='pgdata/sleep-forever'
+if [ -f "${recovery_file}" ]; then
+	set +o xtrace
+	echo "The $recovery_file file is detected, node entered an infinite sleep"
```

Comment:
if you restart a pod that has `/pgdata/sleep-forever`, you can't start PostgreSQL using `pg_ctl -D /pgdata/pg17 start` right away because `/tmp/postgres` does not exist. maybe we should create this directory in this block to make people's life easier. wdyt @hors?

Reply:
I think we can, but only in case if we have sleep file and we don't have this die

Reply:
so @hors you are suggesting that we should create `/tmp/postgres` only if `sleep-forever` file exists and  if `/tmp/postgres` doesn't exist yet, is that correct?

Reply:
since we moved the scripts to the pgo codebase, this was address in the respective PR on the operator.